src/main/java/org/sunbird/workflow/config/Constants.java

```diff
@@ -317,7 +317,7 @@ private Constants() {
 	public static final String TRANSFER_SUBJECT_LINE = "Transfer Notification: #username";
 	public static final String LAST_UPDATED_ON = "lastUpdatedOn";
 	public static final String DOMICILE_MEDIUM = "domicileMedium";
-	public static final String PINCODE = "pinCode";
+	public static final String PIN_CODE = "pinCode";
```

Comment:
Do not rename... it is better to add new one.

Reply:
Done as recommended @karthik-tarento